src/components/vega-wallet/vega-wallet.tsx

```diff
@@ -118,6 +147,29 @@ const VegaWalletNotConnected = () => {
   );
 };
 
+interface VegaWalletAssetsListProps {
+  accounts: WalletCardAssetProps[];
+}
+
+const VegaWalletAssetList = ({ accounts }: VegaWalletAssetsListProps) => {
+  const { t } = useTranslation();
+  if (!accounts.length) {
+    return null;
+  }
+  return (
+    <>
+      <WalletCardHeader>
+        <BulletHeader style={{ border: "none" }} tag="h2">
+          {t("assets")}
+        </BulletHeader>
+      </WalletCardHeader>
+      {accounts.map((a) => (
```

Comment:
is there not a warning about key missing?

Reply:
there is it's at the bottom of the file 👍